packages/theme/src/checkbox/responsive.less

```
@@ -0,0 +1,18 @@
+@import '../mixins/common.less';
+@import '../mixins/checkbox.less';
+@import '../custom.less';
+@import './vars.less';
+
+@checkbox-prefix-cls: ~'@{css-prefix}checkbox';
+
+@media screen and (max-width: 1280px) {
+  .@{checkbox-prefix-cls} {
```

Stylelint (17.3.0)

[error] Unknown word checkbox-prefix-cls (CssSyntaxError)

(CssSyntaxError)